src/Cache/FixedTaggingCachePool.php

```diff
@@ -97,7 +97,9 @@ public function deleteItems(array $keys)
      */
     public function save(CacheItemInterface $item)
     {
-        $this->addTags($item);
+        if ($item instanceof TaggableItemInterface) {
```

Comment:
`TaggableItemInterface` is deprecated. We should move towards `TaggableCacheItemInterface`.

I will merge this and the look over the bundles to see what changes we need to do for the latest version of tags.